packages/gestalt-datepicker/src/DateField/InternalDateField.js

```diff
@@ -263,6 +285,17 @@ function InternalDateField({
     }
```

Comment:
I think we should split out into separate functions for readability

```
let translations = getTranslationsFromMUIJS(code);
 if(!locale) {
  translations = getLocalTranslations(code)
 }
```

Reply:
DONE